### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by StelFux**

### CODE REVIEW

File: po/fr_FR.UTF-8.po [hunk 1]
```diff
@@ -0,0 +1,158 @@
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: lutris\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (stephanlachnit) [inline po/fr_FR.UTF-8.po:5]: Missing

File: po/LINGUAS [hunk 2]
```diff
@@ -0,0 +1 @@
+fr_FR.UTF-8
```
Comment (stephanlachnit) [inline po/LINGUAS:1]: just call the file `fr`

File: po/fr_FR.UTF-8.po [hunk 3]
```diff
@@ -0,0 +1,158 @@
+#, fuzzy
```
Comment (stephanlachnit) [inline po/fr_FR.UTF-8.po:1]: Remove this line.

File: po/fr_FR.UTF-8.po [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+msgid ""
+msgstr ""
+"Project-Id-Version: lutris\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-21 14:50+0200\n"
+"PO-Revision-Date: 2020-04-20 12:40+0200\n"
+"Last-Translator: Stelfux <coles@tuta.io>\n"
+"Language-Team: French - France <>\n"
```
Comment (stephanlachnit) [inline po/fr_FR.UTF-8.po:9]: Just call it `French \n` since there is no link.

File: po/fr_FR.UTF-8.po (hunk 1 above)
Comment (StelFux) [reply po/fr_FR.UTF-8.po]: The first two lines are necessary for the execution of `ninja lutris-update-po`.

File: po/fr.po [hunk 5]
```diff
@@ -0,0 +1,153 @@
+"POT-Creation-Date: 2020-04-21 14:50+0200\n"
+"PO-Revision-Date: 2020-04-22 12:40+0200\n"
+"Last-Translator: Stelfux <coles@tuta.io>\n"
+"Language-Team: French\n"
+"Language: fr_FR\n"
```
Comment (stephanlachnit) [inline po/fr.po:5]: `fr`

**Pull request merged**